### PR TITLE
Add jellyfin authentication

### PR DESCRIPTION
**Pull request by JVT038**

### CODE REVIEW

File: public/js/settings-integration-plex.js
Comment (JVT038) [inline public/js/settings-integration-plex.js:1]: What is this?

Comment (leepeuker) [reply public/js/settings-integration-plex.js:1]: I reverted this, was wrong

**Comment by leepeuker** [comment]
Notes:
- improve connection test message, differentiate between server url and auth success/errors
- improve error messages in general

**Comment by leepeuker** [comment]
I was thinking how we could improve the `Test connection` feature. Maybe we could test for two separate things:
- that the server url exists and can be connected to
- that the authentication is correct

Resulting alerts of the test:
- server url does not exist: display error ("Server exists but authentication not set or invalid")
- server url does exist but auth does not wotk/exist: display warning ("Server exists but authentication not set or invalid")
- server url does exist and auth does work: success

this way the user can get independent results for the server url and the auth :thinking:

**Review by leepeuker — APPROVED** [review]

**Pull request merged**